lnrpc/invoicesrpc/invoices.proto

```diff
@@ -82,6 +82,11 @@ message AddHoldInvoiceRequest {
 
     /// Whether this invoice should include routing hints for private channels.
     bool private = 9 [json_name = "private"];
+
+    /**
+    The list of private channel ids that will be used as hop hints.
```

Comment:
nit: add that these are short channel IDs?
It's apparent from the uint64 when you're used to the rpc, but might be helpful for newer users :)